en/docs/guides/Provisioning/role-based-provisioning.md

```diff
@@ -0,0 +1,17 @@
+# Configure Role-Based Provisioning
+
+Here, we discuss how to provision users based on the roles they are assigned. In role-based provisioning, the user is provisioned when the user is added to a preconfigured role, and the user is deleted from the trusted identity provider, when the user is removed from the role.
+
```

Comment:
Ideally, this explanation should not be required here and we can simply link to a concepts page

Reply:
same comments for role based provisioning

Reply:
Added this as a one-liner. We don't really need to point it off to concepts sections before following the guide IMO, since the objective is obvious.